### ISSUE TITLE
Elements that have scrollable content under the "About" tab is not accessible by keyboard.

### ISSUE THREAD

**Issue by Xiaoyu-Huang** [Azure/LogicAppsUX#7280]
### Describe the issue

Repro Steps:

Go to any workflow, any action, go to about tab, the scrollar  bar is not accessible. I tested in Azure portal, the issue could be reproed

![Image](https://github.com/user-attachments/assets/833e4f34-0c10-4d22-957c-0e9e153d7d9f)

### Are you using new designer or old designer

New Designer (With action panels)

### Browser

Edge
